server/handlers_v2.go

```diff
@@ -271,23 +278,45 @@ func (server HTTPServer) getRecommendations(writer http.ResponseWriter, request
 	}
 }
 
-func getRecommendationsFillImpacted(
+func generateRuleAckMap(acks []types.SystemWideRuleDisable) (ruleAcksMap map[types.RuleID]bool) {
+	ruleAcksMap = make(map[types.RuleID]bool)
```

Comment:
this is nitpick, but usually `{}interface` is used instead of concrete data type to create set

Reply:
you mean like `make(map[types.RuleID]interface{})`?
But that cannot be returned in the function as `map[types.RuleID]bool`

Reply:
yes, it would be map of (empty) structures basically. But it is really really minor